### PR TITLE
1 feature monitor and message receiver

### LINKED ISSUE #1: Feature - Monitor and message receiver
- Program 1 (Main monitor): Monitoring of newly mined Bitcoin block height and hash
  - When a new Bitcoin block is found, the program sent the block height and hash string to the message receiver.

- Program 2 (Message receiver): Waiting for a message after connecting to the message queue
  - Output to standard output when a message for a new block arrives (class object sample)

### PR DESCRIPTION
**Pull request by Yujin-nKim**

### CODE REVIEW

**Comment by munhyunsu** [comment]
Things I think should be fixed.

1. Each file must end with `breakline`. (File coding convention)
2. The standard library of `import` and the 3rd-party library should be separated by one breakline. (Coding convention)
3. Functions (`def`) and `import`, `if __name__ == '__main__'` should be separated by two breaklines. (Coding convention)
4. In the `if __name__ == '__main__'` division, it should be designed in advance so that argument parsing using argparse is possible.
4-2. In particular, since these two codes are the main process, `parameter parsing' must be enabled.
4-3. [Note](https://github.com/munhyunsu/Hobby/blob/master/StarterCode/main.py)
5. `test.py` contains the account name and password. You need to change it to secret.py and import the file to use it.
5-2. The password has been leaked online, so you need to change your password!
6. Both `message_receiver.py` and `Monitor.py` should handle graceful exit with `try-except KeyboardInterrupt`
7. All file names need to be modified. (Coding convention)
7-2. Suggestion: `main_monitor.py` `main_blockhandler.py`
8. Requires `requirements.txt` what 3rd-party library is required!

- Ref [Google Python Style Guide](http://eiichiroi.github.io/google-styleguide-ja/pyguide.html)

File: Monitor.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import time
+import zmq
+
+from bitcoinrpc.authproxy import AuthServiceProxy, JSONRPCException
+
+#가장 최근 블록의 해시값 리턴
+def monitor():
+    rpc_connection = AuthServiceProxy("http://%s:%s@127.0.0.1:8332" % ("yujin", "dnlab2022"))
```
Comment (munhyunsu) [inline Monitor.py]: Personal information exposure code: Need to use `secret.py`

File: Monitor.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    while(True):
+        value = monitor()
+        time.sleep(3)
+        best_block_hash = monitor()
+        if(value != best_block_hash):
+            message = monitor_message(best_block_hash)
+            socket.send_json(message)
+            print('New Block : ', message)
+
```
Comment (munhyunsu) [inline Monitor.py]: unnecessary breaklines

**Review by munhyunsu — CHANGES_REQUESTED** [review]
Things I think should be fixed.

1. Each file must end with `breakline`. (File coding convention)
2. The standard library of `import` and the 3rd-party library should be separated by one breakline. (Coding convention)
3. Functions (`def`) and `import`, `if __name__ == '__main__'` should be separated by two breaklines. (Coding convention)
4. In the `if __name__ == '__main__'` division, it should be designed in advance so that argument parsing using argparse is possible.
4-2. In particular, since these two codes are the main process, `parameter parsing' must be enabled.
4-3. [Note](https://github.com/munhyunsu/Hobby/blob/master/StarterCode/main.py)
5. `test.py` contains the account name and password. You need to change it to secret.py and import the file to use it.
5-2. The password has been leaked online, so you need to change your password!
6. Both `message_receiver.py` and `Monitor.py` should handle graceful exit with `try-except KeyboardInterrupt`
7. All file names need to be modified. (Coding convention)
7-2. Suggestion: `main_monitor.py` `main_blockhandler.py`
8. Requires `requirements.txt` what 3rd-party library is required!

- Ref [Google Python Style Guide](http://eiichiroi.github.io/google-styleguide-ja/pyguide.html)

**Comment by Yujin-nKim** [comment]
수정완료했습니다.

**Review by munhyunsu — APPROVED** [review]

**Pull request merged**